src/WebSocketsClient.cpp

```diff
@@ -242,7 +244,12 @@ void WebSocketsClient::loop(void) {
         if(_client.isSSL) {
             DEBUG_WEBSOCKETS("[WS-Client] connect wss...\n");
             if(_client.ssl) {
-                delete _client.ssl;
+                #if defined(WIO_TERMINAL) || defined(SEEED_XIAO_M0) || defined(ARDUINO_SAMD_MKRWIFI1010) || defined(ARDUINO_SAMD_NANO_33_IOT)

```

Comment:
better use the `WEBSOCKETS_NETWORK_TYPE == NETWORK_SAMD_SEED || WEBSOCKETS_NETWORK_TYPE == NETWORK_WIFI_NINA`